pyproject.toml

```diff
@@ -17,6 +17,7 @@ Sphinx = "^7.2"
 pytest = "^7.4"
 mypy = "^1.8"
 types-Pillow = "^10.1"
+roman = "^4.1"
```

Comment:
You want this as a regular dependency (along with Pillow above) rather than a dev dependency.

Reply:
Oops - so I do! Thanks